mobileversion.html

```diff
@@ -227,5 +227,21 @@ <h1 class="Skills">Skills <i class="fas fa-caret-down"></i></h1>
         </div>
         </div>
      </section>
-      </body>
+     <footer class="contact-form">
+      <h1 class="contactme">Contact me</h1>
+         <p class="app">
+            If you have an application you<br> 
+            are interested in developing, a <br>
+            feature that you need built or a <br>
+            project that needs coding. I’d <br>
+            <span>love to help with it </span>
+         </p>
+         <form class="form1" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- [ ] [OPTIONAL] I noticed that we can not collect the data from your form because you did not implement correctly the [Formspree service](https://formspree.io/html). This is  `https://formspree.io/f/{form_id}` is just an example. I am sharing a recorded video with you in order for you to solve this issue. I strongly recommend you implement it because it is a key requirement for this project. Please fix it :smile: 
https://www.loom.com/share/7447923120c8413a9fd14e0f58e5ea03